src/matrix.jl

```diff
@@ -48,9 +48,23 @@ The matrix norm of an interval matrix.
 - `A` -- interval matrix
 - `p` -- (optional, default: `Inf`): the class of `p`-norm
 """
-function LinearAlgebra.norm(A::IntervalMatrix, p::Real=Inf)
+function LinearAlgebra.opnorm(A::IntervalMatrix, p::Real=Inf)
     if p == Inf
-        return LinearAlgebra.norm(max.(abs.(left(A)), abs.(right(A))), Inf)
+        # for p=Inf, the operator norm is the maximum absolute row sum
+        res = 0
+        for i in axes(A, 1)
+            res = max(res, sum(max(abs(itv.left), itv.right)
```

Comment:
seems to me that `abs` should also be applied as well to the right part of `itv`, or not?